gravitee-am-ui/src/app/domain/settings/theme/theme.component.ts

```diff
@@ -119,10 +286,48 @@ export class DomainSettingsThemeComponent implements OnInit {
       this.themeService.create(this.domain.id, themeToPublish) :
       this.themeService.update(this.domain.id, this.theme.id, themeToPublish);
 
-    publishAction.subscribe(() => {
+
+    this.selectedForm.content = this.selectedTemplateContent;
+    const formAction = (!this.selectedForm.id) ?
+      this.formService.create(this.domain.id, null, this.selectedForm, false) :
+      this.formService.update(this.domain.id, null, this.selectedForm.id, this.selectedForm, false);
+
+    forkJoin([publishAction, formAction]).subscribe(() => {
```

Comment:
@tcompiegne , calling `formAction` throws exception "content: must be null" exception if `this.selectedForm.content` is undefined in line 290. `this.selectedTemplateContent` will be undefined in case user only provides theme data (logo width/url) and press Publish button without selecting any custom form. 
creating `formAction` if `this.selectedTemplateContent` not undefined fixes the issue.